frontend/server/src/main/resources/proto/inference.proto

```diff
@@ -15,11 +16,20 @@ message PredictionsRequest {
 
     // Input data for model prediction
     map<string, bytes> input = 3; //required
+
+    // SequenceId is required for StreamPredictions2 API.
+    optional string sequence_id = 4; //optional
```

Comment:
This id needs to be unique. How to you ensure that two clients do not sent the same id? Should this not be generated on the server? We're streaming in inputs through the same connection anyways. Why does the client need to know about the sequence_id anyways?

Reply:
The sequence id is similar as streaming message topic. A client can reuse the sequence id and continue generating results using backend caching when the client lose the connection. It is a little bit complicate if it is generated by server side since the id needs to be returned to client side for the next time reuse when the client lose the connection.

Reply:
Connection loss would not mean total loss of memory on client so he could still use the id. In any case we should document this in the README to ensure users are aware of this.